info/container.go

```diff
@@ -405,3 +405,34 @@ func calculateCpuUsage(prev, cur uint64) uint64 {
 	}
 	return cur - prev
 }
+
+type Percentiles struct {
+	// Average over the collected sample.
+	Mean uint64 `json:"mean"`
+	// Max seen over the collected sample.
+	Max uint64 `json:"max"`
+	// 90th percentile over the collected sample.
+	Ninety uint64 `json:"ninety"`
+}
+
+type Usage struct {
+	// Indicates amount of data available [0-100].
+	// If we have data for half a day, we'll still process DayUsage,
+	// but set PercentComplete to 50.
+	PercentComplete int32 `json:"percent_complete"`
+	// Mean, Max, and 90p cpu rate value in milliCpus/seconds. Converted to milliCpus to avoid floats.
+	CpuUsage Percentiles `json:"cpu_usage"`
```

Comment:
How about map[ResourceName]ResourceQuantity instead of explicit fields? Take a look at [type quantity](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/resource/quantity.go#L86) that is part of kubernetes. That might help avoiding having to infer scaling factors from comments or json description.